ui/app/templates/components/role-aws-edit.hbs

```diff
@@ -52,11 +52,15 @@
       <MessageError @model={{this.model}} />
       <NamespaceReminder @mode={{this.mode}} @noun="AWS role" />
       {{#if (gt this.model.credentialTypes.length 1)}}
-        <AlertBanner
-          @type="warning"
-          @message="This role has more than one credential_type, currently: {{join ', ' this.model.credentialTypes}}.
-            Multiple credential types is deprecated and you must choose one in order to save this role."
-        />
+        <Hds::Alert @type="inline" @color="warning" class="has-bottom-margin-s" as |A|>
+          <A.Title>Warning</A.Title>
+          <A.Description>
+            This role has more than one
+            <code>credential_type</code>, currently:
```

Comment:
thanks for adding this code block!

Reply:
thanks for noticing! I thought it was a nice touch! ✨